tests/e2e/default/service-worker/service-worker.wdio-spec.js

```diff
@@ -105,6 +93,11 @@ describe('Service worker cache', () => {
     await loginIfNeeded();
   });
 
+  after(async () => {
+    await utils.deleteUsers([chw]);
+    await utils.revertDb([/^form:/], true);
+  });
+
   afterEach(async () => {
     await utils.revertSettings(true);
```

Comment:
Is this necessary in afterEach? L198 is also doing a revertSettings, and I wonder if every test needs to revert.

Reply:
good catch! ⭐